paddle/fluid/operators/gaussian_random_op.cc

```diff
@@ -58,12 +86,18 @@ class GaussianRandomOp : public framework::OperatorWithKernel {
     for (auto dim : shape) {
       temp.push_back(static_cast<int64_t>(dim));
     }
-    PADDLE_ENFORCE_GT(
-        shape.size(), 0UL,
-        platform::errors::InvalidArgument(
-            "Attribute(shape) of GaussianRandomOp must be set "
-            "and shape.size() > 0, but reveived shape.size() is %d",
-            shape.size()));
```

Comment:
Better not to delete this `PADDLE_ENFORCE_GT `

Reply:
Done